pyracf/common/downstream_fatal_error.py

```diff
@@ -0,0 +1,66 @@
+"""Exception to use when IRRSMO00 returns with an error."""
+from typing import Union
+
+
+class DownstreamFatalError(Exception):
+    """
+    Raised IRRSMO00 returns with a return code of 8, indicating an error occured prior to request.
+    """
+
+    def __init__(
+        self,
+        xml_str: str,
+        request_xml: bytes,
+        run_as_userid: Union[str, None] = None,
+        result_dict: dict = None,
```

Comment:
Change `result_dict` to `result_dictionary`.

Reply:
fixed.